app/src/main/java/com/example/pokedex/domain/Pokemon.kt

```diff
@@ -0,0 +1,33 @@
+package com.example.pokedex.domain
+
+data class Pokemon(
```

Comment:
Are you using this? I couldn't find anywhere this was used - but maybe this is still work-in-progress?

Reply:
WIP